taskcluster/utils.py

```diff
@@ -1,11 +1,11 @@
 import re
 import json
 import datetime
-import uuid
 import base64
 import logging
 import os
 import requests
+import slugid as slugid_
```

Comment:
Ah yes, my mistake! The function defined below is `slugId` not `slugid` so there was no collision. Good spot. :+1: